src/adc/iio/hardware.rs

```diff
@@ -82,6 +82,8 @@ const CHANNELS_PWR: &[ChannelDesc] = &[
     ("current", "powerboard-factory-data/pwr-curr", "pwr-curr"),
 ];
 
+const TIMESTAMP_ERROR: u64 = u64::MAX;
```

Comment:
As discussed: please annotate why the program will never reach `u64::MAX` during its runtime and it is therefore safe to use this magic value.